src/solvers/lp_solvers.rs

```diff
@@ -54,6 +54,33 @@ impl<T: lp_solvers::solvers::SolverTrait + Clone> Solver for LpSolver<T> {
     }
 }
 
+impl<T> crate::solvers::WithMipGap for Model<T>
+where
+    T: lp_solvers::solvers::WithMipGap<T>,
+{
+    fn mip_gap(&self) -> Option<f32> {
+        self.solver.mip_gap()
+    }
+
+    fn with_mip_gap(mut self, mip_gap: f32) -> Result<Self, MipGapError> {
```

Comment:
Maybe we should add a doc comment here with one or two sentences about what the MIP gap is ?